Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -778,19 +778,20 @@ - (NSMutableDictionary *)persistenceDict {
             NSDictionary *persistenceDict = nil;
             @try {
                 NSError *error = nil;
-                NSData *data = [NSData dataWithContentsOfURL:self.class.URLForPrefsFile
-                    options:0 error:&error];
-                if (!error && data)
+                NSData *data = [NSData dataWithContentsOfURL:self.class.URLForPrefsFile options:0 error:&error];
```

Comment:
it would be better to replace `0` with `NSDataReadingMappedIfSafe`

Reply:
Probably, I didn't update the existing code.  Trying to resist the temptation to refactor a bunch of this.